feature/gnoi/os/tests/osinstall/osinstall_test.go

```diff
@@ -257,29 +224,49 @@ func (tc *testCase) transferOS(ctx context.Context, t *testing.T, standby bool)
 // verifyInstall validates the OS.Verify RPC returns no failures and version numbers match the
 // newly requested software version.
 func (tc *testCase) verifyInstall(ctx context.Context, t *testing.T) {
-	r, err := tc.osc.Verify(ctx, &ospb.VerifyRequest{})
-	if err != nil {
-		t.Fatalf("OS.Verify request failed: %s", err)
-	}
+	deadline := time.Now().Add(*timeout)
+	for time.Now().Before(deadline) {
+		r, err := tc.osc.Verify(ctx, &ospb.VerifyRequest{})
+		switch status.Code(err) {
+		case codes.OK:
+		case codes.Unavailable:
+			t.Log("Reboot in progress.")
+			time.Sleep(time.Minute)
```

Comment:
this duration of time.Minute is reused multiple times in this func, if this is intended to be the standard wait time for reboots or other conditions that desire a sleep, then create a var for this duration and reference the var in these locations in case it is deemed appropriate to change the standard wait time in the future

Reply:
Sounds good, made the changes locally, just firing a test-run on my DUT, upon success I'll raise a new commit.

Reply:
This var does not need to be exported, also prefer `rebootWait` or similar to avoid renaming in case of increase to wait time.

Other than that this looks good to me for golang

Reply:
Sure, I actually had it as `rebootWait` at first, but thought it was a bit misleading. Let me do those quick changes, rebase and push.